main.go

```diff
@@ -15,11 +15,11 @@ const (
 )
 
 type Parcel struct {
-	Number    int
-	Client    int
-	Status    string
-	Address   string
-	CreatedAt string
+	Number     int
+	Client     int
+	Status     string
+	Address    string
+	Created_At string
```

Comment:
В как раз принято использовать PascalCase
```suggestion
	CreatedAt string
```

Reply:
Приветствую Иван, в прекоде было CreatedAt, я и не думал что мы должны писать иначе, или менять прекод! но мне пришлось это сделать, просто потому, что в базе столбец указан Created_At! получается в исходниках есть ошибка! 
![9](https://github.com/user-attachments/assets/e5b3eab5-507e-4f4e-b635-fbde7716a55e)

Reply:
Как раз сейчас расписывал все Наталье

Reply:
![image](https://github.com/user-attachments/assets/a49040e1-e11f-4fe4-b70e-6f8e50e5bf74)

Reply:
Вот последний скрин крупно
![image](https://github.com/user-attachments/assets/8f136e7e-7009-4456-942b-8e8c34d11ac1)